### PR TITLE
[TESTING/WIP] v1.17 ci breakage

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: examples/cache/front-envoy.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ static_resources:
             typed_config:
               "@type": "type.googleapis.com/envoy.extensions.filters.http.cache.v3alpha.CacheConfig"
               typed_config:
-                "@type": "type.googleapis.com/envoy.source.extensions.filters.http.cache.SimpleHttpCacheConfig"
+                "@type": "type.googleapis.com/envoy.extensions.cache.simple_http_cache.v3alpha.SimpleHttpCacheConfig"
```
Comment (antoniovicente) [inline examples/cache/front-envoy.yaml:34]: My suggestion would be to simply delete this example file rather than backport the API change.

Comment (phlax) [reply examples/cache/front-envoy.yaml:34]: let me try and figure out why its happening - im pretty not keen on this approach either 

im also reluctant to delete the example as it matches to the related docs

Comment (phlax) [reply examples/cache/front-envoy.yaml:34]: yep - this isnt going to work easily - ill use this pr to debug failing test...

**Comment by antoniovicente** [comment]
What are the next steps for this PR?

/wait

**Comment by phlax** [comment]
> What are the next steps for this PR?

i think the fix is in #17080 so ill close it

**Pull request closed without merging**